.github/workflows/patch.js

```diff
@@ -49,7 +49,8 @@ module.exports = async ({ context, github, core }) => {
   });
   const version = latestRelease.data.tag_name.replace("v", "");
   const [major, minor, micro] = version.split(".");
-  const label = `patch-${major}.${minor}.${parseInt(micro) + 1}`;
+  const nextMicro = version.includes("rc") ? micro : (parseInt(micro) + 1).toString();
+  const label = `patch-${major}.${minor}.${nextMicro}`;
```

Comment:
Examples:

- The next patch release of `2.x.0rc0` is `2.x.0`.
- The next patch release of `2.x.0` is `2.x.1`.